hybrid/SampleApps/NoteSync/src/com/salesforce/samples/notesync/ContentSoqlSyncTarget.java

```diff
@@ -223,7 +225,16 @@ else if (inResults && parser.getName().equals(SIZE)) {
                             totalSize = Integer.parseInt(parser.nextText());
                         }
                         else if (inRecord && parser.getName().startsWith(SF)) {
-                            record.put(parser.getName().substring(SF.length()), parser.nextText());
```

Comment:
Record type is under the "type" field with soap instead of being under attributes / type (different from soql and sosl responses)